TOC.md

```diff
@@ -248,6 +248,18 @@
   + 系统数据库
     - [`mysql`](/reference/system-databases/mysql.md)
     - [`information_schema`](/reference/system-databases/information-schema.md)
+    + [`sql-diagnosis`](/reference/system-databases/sql-diagnosis.md)
+      -[`cluster_info`](/reference/system-databases/cluster-info.md)
+      -[`cluster_hardware`](/reference/system-databases/cluster-hardware.md)
+      -[`cluster_config`](/reference/system-databases/cluster-config.md)
+      -[`cluster_load`](/reference/system-databases/cluster-load.md)
+      -[`cluster_systeminfo`](/reference/system-databases/cluster-systeminfo.md)
+      -[`cluster_log`](/reference/system-databases/cluster-log.md)
+      -[`metrics_schema`](/reference/system-databases/metrics-schema.md)
+      -[`metrics_tables`](/reference/system-databases/metrics-tables.md)
+      -[`metrics_summary`](/reference/system-databases/metrics-summary.md)
+      -[`inspection_result`](/reference/system-databases/inspection-result.md)
+      -[`inspection_summary`](/reference/system-databases/inspection-summary.md)
```

Comment:
This section has a display error because lack of one space. Preview this file and you'll see.
```suggestion
      - [`cluster_info`](/reference/system-databases/cluster-info.md)
      - [`cluster_hardware`](/reference/system-databases/cluster-hardware.md)
      - [`cluster_config`](/reference/system-databases/cluster-config.md)
      - [`cluster_load`](/reference/system-databases/cluster-load.md)
      - [`cluster_systeminfo`](/reference/system-databases/cluster-systeminfo.md)
      - [`cluster_log`](/reference/system-databases/cluster-log.md)
      - [`metrics_schema`](/reference/system-databases/metrics-schema.md)
      - [`metrics_tables`](/reference/system-databases/metrics-tables.md)
      - [`metrics_summary`](/reference/system-databases/metrics-summary.md)
      - [`inspection_result`](/reference/system-databases/inspection-result.md)
      - [`inspection_summary`](/reference/system-databases/inspection-summary.md)
```